clash-cores/src/Clash/Cores/Xilinx/Ethernet/Gmii.hs

```diff
@@ -0,0 +1,136 @@
+{-|
+  Copyright   :  (C) 2024, Google LLC
+  License     :  BSD2 (see the file LICENSE)
+  Maintainer  :  QBayLogic B.V. <devops@qbaylogic.com>
+
+ Contains a wrapper for the Xilinx GMII to SGMII PMA core :
+  "LogiCORE IP Ethernet 1000BASE-X PCS/PMA or SGMII"
+
+-}
+
+{-# LANGUAGE BangPatterns #-}
+{-# LANGUAGE RecordWildCards #-}
+{-# LANGUAGE ViewPatterns #-}
+{-# OPTIONS_GHC -fconstraint-solver-iterations=10 #-}
+
+module Clash.Cores.Xilinx.Ethernet.Gmii where
+
+import Clash.Explicit.Prelude
+
+import Clash.Annotations.Primitive
+import Clash.Cores.Xilinx.Ethernet.Gmii.BlackBoxes
+import Clash.Cores.Xilinx.Ethernet.Gmii.Types
+import Clash.Explicit.Prelude
+import Clash.Signal.Internal(ClockN, DiffClock(..))
+
+-- | Wrapper for the LogiCORE IP Ethernet 1000BASE-X PCS/PMA or SGMII.
+-- Configured to function as a GMII to SGMII bridge.
+gmiiSgmiiBridge ::
+  forall asyncDom sgmii625 gmiiRx125 gmiiTx125 .
+  ( KnownDomain asyncDom
+  , KnownDomain sgmii625
+  , KnownDomain gmiiRx125
+  , KnownDomain gmiiTx125
+  , DomainPeriod sgmii625 ~  Picoseconds 1600
+  , DomainPeriod gmiiRx125 ~ Nanoseconds 8
+  , DomainPeriod gmiiTx125 ~ Nanoseconds 8
+  , DomainActiveEdge sgmii625 ~ Rising
+  , DomainActiveEdge gmiiRx125 ~ Rising
+  , DomainActiveEdge gmiiTx125 ~ Rising
+  ) =>
+  -- | Reference clock coming from the Phy.
```

Comment:
```suggestion
  -- | Reference clock coming from the PHY
```

This is in more places: you write MAC all-caps, which I agree with, but you don't do it for PHY.